<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>770</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Impact</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
background-color:rgb(255, 255, 255);
}

QLabel
{
background-color:rgb(255, 255, 255);
border: 2px solid rgb(0, 0, 0);
}

QTableWidget
{
color: rgb(0, 0, 0);
background-color:rgb(113, 107, 150);
border: 0px solid #706087;
}

QTableWidget::item
{
background-color:rgb(112, 96, 135);
}

QHeaderView::section
{
background-color:rgb(112, 96, 135);
}

QTableWidget QTableCornerButton::section {background-color: rgb(112, 96, 135); }

QLineEdit
{
color: rgb(0, 0, 0);
background-color:#ffffff;
border: 3px solid #000000;
border-radius: 8px;
padding: 0 2px;
selection-background-color:rgb(160, 152, 205);

}

QLineEdit:!focus
{
border:3px solid #000000; 

}

QPushButton
{
color: rgb(0, 0, 0);
background-color:#ffffff;
border: 4px solid #000000;
padding: 0 2px;
}

QPushButton:pressed
{
color: rgb(0, 0, 0);
background-color:#ffffff;
border: 3px solid #000000;
padding: 0 2px;
}

QCheckBox
{
background-color:rgb(150, 93, 135);
border: 4px solid rgb(116, 72, 105);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <family>Impact</family>
    </font>
   </property>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>690</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>690</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_path">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>610</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_way">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>650</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>650</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Путь:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>610</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Длина пути:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>690</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Impact</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начальный город:</string>
    </property>
   </widget>
   <widget class="QLabel" name="graf">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>561</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
